design/src/high-level/genre-analysis.md

```diff
@@ -13,67 +13,121 @@
 
 ## Defining characteristics
 
-- automated resource extraction
-- automation resource refinement
-- logistical challenges
-- micro-optimizations
+- automated resource extraction and refinement
+- logistical challenges driven by spatial constraints
+- optional optimization of existing systems (generally optimizing resource throughput)
 - gated technology progression
+- problems/puzzles/modules flow into each other (as opposed to puzzle games or zachlikes where they are explicitly independent)
+- partially self-directed goals
+- challenge can be overcome by spending time waiting
+- highly moddable
+- procedurally generatd world
+
+## Why is this genre fun?
+
+- puzzle like, but complete solutions are easy to achieve
+- optional optimizations on top of said solutions
+- satisfaction of watching your contraptions run
+- satisfaction of taming the wild
+- interesting, unexpected failures due to unexpected interactions or subtle bugs
+- gameplay can be extended, enhanced, or changed through mods or built in options
+- hits on "engineering creativity" in a way that other games don't
+  - engineering creatvitiy is more about coming up with solutions within constraints. It's not a blank canvas. It's based on analytically evaluable metrics
+- scratches similar itches to programming, but with clearer gradients and fewer frustrations
+  - great debugging: can see what's working and why with exceptional visualization
+  - basic solutions are straightforward
+  - very weak syntax constraints, very intuitive syntax
+  - solutions generally do not need to be as robust as in the real world
+  - tech progression doesn't break previous (natural) assumptions
+  - good quality of life tools and docs
 
 ## Adjacent Genres
 
-- Base and city builders
-  - Build out a complex and delightful area, but with no / little automation
+This list is roughly ordered by the closeness to the genre.
+
 - Zachlike puzzles (automation heavy puzzle games)
+  - Opus Magnum, Incredible machine
+  - Greater focus on additional (seconday) optimization constriants (which are provided by the game)
   - Micro-optimization as an entire game
+- Base and city builders
+  - Sim City, The Sims, Cities: Skylines, Dragon Quest Builders
+  - Build out a complex and delightful area, but with no / little automation
+  - Most of the actual functioning of your constructions is abstracted away
 - Colony sims
+  - Rimworld, Dwarf Fortress, Oxygen Not Included
   - Balance complex systems, but with a focus on the needs of your workers, not scaling production
+  - More about the emergent push and pull of life in the colony. Less about steadily pushing towards some goal.
 - Idle games
+  - Forager, Cookie Clicker, Candy box
```

Comment:
Yeah, it's close to Stardew Valley with more idle game elements. I'm unsure if that makes it a better or worse example here.

Reply:
I think we want to keep the examples relatively straightforward, at least for this document. At some point it might be good to compare Forager (and Stardew Valley likes more broadly), but this probably isn't the place. @TimJentzsch, do you have a suggestion for an alternate example? Otherwise I can just cut it :)

Reply:
Nothing that comes to mind right now :)

It might be worth to open a new bullet point for Stardew & co., but honestly I'm not even sure what to call the genre.

Reply:
```suggestion
  - Cookie Clicker, Candy box
```